044_bb_L1118/044_bb_L1118_unsafe.py

```
@@ -0,0 +1,2833 @@
+    :param fn: function in parallelism
+    :param commands: args commands of function
+    :param limit: number of parallel process
+    """
+    global args, catalog_path, logs
+
+    # Start a Pool with "limit" processes
+    pool = Pool(processes=limit)
+    jobs = []
+    necessaries_retries = []
+
+    for command, plog in zip(commands, logs):
```

Ruff (0.14.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)